lib/event_sourcery/rspec/event_store_shared_examples.rb

```diff
@@ -1,66 +1,59 @@
 RSpec.shared_examples 'an event store' do
-  let(:aggregate_id) { SecureRandom.uuid }
+  TestEvent2 = Class.new(EventSourcery::Event)
```

Comment:
It's probably not ideal but they're fairly generic concepts. I'd like to see an overhaul of the entire test suite where we use common domain language for all our specs. I think changing them is outside the scope of this PR though.